From 1040978412d3cb841652715336d531aa75fbba62 Mon Sep 17 00:00:00 2001
From: lintao <lintao@rock-chips.com>
Date: Tue, 9 Sep 2014 10:23:35 +0800
Subject: [PATCH] mmc: fix wrong physical partition definition 	eMMC card can
 support up to 7 partitions, including 2 boot, 1 RPMB and 4 GPs. But
 MMC_NUM_PHY_PARTITION is defined to 6 so we should fix it to 7.

---
 include/linux/mmc/card.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/linux/mmc/card.h b/include/linux/mmc/card.h
index b73027298b3a..cfa28593de91 100644
--- a/include/linux/mmc/card.h
+++ b/include/linux/mmc/card.h
@@ -212,11 +212,12 @@ enum mmc_blk_status {
 };
 
 /* The number of MMC physical partitions.  These consist of:
- * boot partitions (2), general purpose partitions (4) in MMC v4.4.
+ * boot partitions (2), general purpose partitions (4) and
+ * RPMB partition (1) in MMC v4.4.
  */
 #define MMC_NUM_BOOT_PARTITION	2
 #define MMC_NUM_GP_PARTITION	4
-#define MMC_NUM_PHY_PARTITION	6
+#define MMC_NUM_PHY_PARTITION	7
 #define MAX_MMC_PART_NAME_LEN	20
 
 /*
-- 
2.35.3

